### ISSUE TITLE
Why isn't CellHookData class exported?

### ISSUE THREAD

**Issue by udvaritibor** [simonbengtsson/jsPDF-AutoTable#648]
I'm using the following code to draw one cell differently than the others and with an image inside.

I'd like to write a method for the **if statements** to remove the code duplication, but the **CellHookData** class is not exported.

Why?

Or maybe I'm using something in a wrong way :D 

Thanks for the help in advance!

```
didParseCell: data => {
	if (data.section === 'body' && data.column.index === 0)
	{
		data.cell.styles.cellPadding = 0;
	}
},
didDrawCell: data => {
	if (data.section === 'body' && data.column.index === 0)
	{
		const imagePadding = data.row.height * 0.16;
		const imageH = data.row.height - imagePadding * 2;
		const imageW = imageH * imageRatio;
		const drawX = data.cell.x + data.cell.width / 2 - imageW / 2;
		const drawY = data.cell.y + data.cell.height / 2 - imageH / 2;
		doc.addImage(image.data, 'PNG', drawX, drawY, imageW, imageH, this.logoSignature, "FAST", 0);
	}
},

```

**Comment by simonbengtsson** [comment]
It should be exported. If you submit a PR I'll marge it right away. Otherwise I'll fix it before next release.

**Comment by udvaritibor** [comment]
I think I can wait. Thanks for the fast reply! :)

**Comment by simonbengtsson** [comment]
Released in 3.5.4
